core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -199,6 +199,16 @@
    * FACT_FILE_EXT
    */
   public static final String FACT_FILE_EXT = ".carbondata";
+
+  /**
+   * DELETE_DELTA_FILE_EXT
+   */
+  public static final String DELETE_DELTA_FILE_EXT = ".deletedelta";
+
+  /**
+   * UPDATE_DELTA_FILE_EXT
+   */
+  public static final String UPDATE_DELTA_FILE_EXT = FACT_FILE_EXT;
```

Comment:
2 variables are not required. Use FACT_FILE_EXT in place of UPDATE_DELTA_FILE_EXT wherever required